File: src/main/java/io/github/support/terminal/application/domains/core/api/telegram/requests/SendMessageRequest.java
package io.github.support.terminal.application.domains.core.api.telegram.requests;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;
import lombok.ToString;
import lombok.experimental.Accessors;

@Data
@ToString
@Accessors(chain = true)
public class SendMessageRequest {

    @JsonProperty("chat_id")
    private Integer chatId;

    @JsonProperty("text")
    private String text;

    @JsonProperty("parse_mode")
    private String parseMode;

    @JsonProperty("reply_to_message_id")
    private String replyToMessageId;

}

File: src/main/kotlin/io/github/bot/terminal/application/domains/integrations/telegram/requests/SendMessageRequest.kt
package io.github.bot.terminal.application.domains.integrations.telegram.requests

import com.fasterxml.jackson.annotation.JsonProperty

class SendMessageRequest {
    @JsonProperty("chat_id")
    private val chatId: Int? = null

    @JsonProperty("text")
    private val text: String? = null

    @JsonProperty("parse_mode")
    private val parseMode: String? = null

    @JsonProperty("reply_to_message_id")
    private val replyToMessageId: String? = null
}